pkgs/development/python-modules/pwinput/default.nix

```diff
@@ -0,0 +1,29 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  pytest,
+  lib,
+}:
+
+buildPythonPackage rec {
+  pname = "pwinput";
+  version = "1.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-yhqL0G4ohy11Hb1BMthjcSfCW0COo6NJN3MUpUkUJvM=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+  propagatedBuildInputs = [ pytest ];
```

Comment:
https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-parameters

Also, shouldn't pytest be in `nativeCheckInputs`, but what's the purpose for it being here if `doCheck` is set to `false`?

Reply:
Yeah, I forgot about thatl.